misk/src/main/kotlin/misk/web/actions/NotFoundAction.kt

```diff
@@ -21,10 +21,16 @@ class NotFoundAction : WebAction {
   @ResponseContentType(MediaTypes.ALL)
   @Unauthenticated
   fun notFound(@PathParam path: String): Response<ResponseBody> {
-    return Response(
-        body = "Nothing found at /$path".toResponseBody(),
-        headers = Headers.of("Content-Type", MediaTypes.TEXT_PLAIN_UTF8),
-        statusCode = 404
-    )
+    return response(path)
+  }
+
+  companion object {
```

Comment:
curious: why a companion? I assume you’re gonna use this elsewhere

Reply:
Currently use it from StaticResourceAction to ensure consistent 404 responses